docs/releases/3.1.txt

```diff
@@ -162,6 +162,11 @@ Models
 
 * ...
 
+Pagination
+~~~~~~~~~~
+
+* Support for iterating over ``Paginator`` was added.
```

Comment:
was -> is
(or something like "Paginator can now be iterated over to yield its pages.")
(all release notes in present tense)

Reply:
OK, thanks Tim. I shall do that in the morning now. 👍